### PR TITLE
Feature branch: header and hero section

### PR DESCRIPTION
**Pull request by jash-debug**
Add the header section
Add The hero section
Add descriptive readme
Set Hmtl and CSS structure

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+<body>
+    <nav id="nav">
+        <div class="nav">
+            <a class="nav__logo" href="#">My Logo</a>
+
+            <img class="nav__hambuger" src="images/hambuger.svg" alt="Icon">
+
+        </div>
+    </nav>
```
Comment (Ismailco) [inline index.html]: - [x] The container of this section should be a header tag, kindly wrap your `nav` tag in a `header` tag so your page has a header section

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                Glad to see you!
+            </h1>
+            <p class="hero__desc">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <p class="hero__subtitle">
+                LET’S CONNECT
+            </p>
```
Comment (Ismailco) [inline index.html]: - [x] [OPTIONAL] It would be better if you used a `<a>` tag for this link so you can refer to an email or external website

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @jash-debug, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jash-debug) [reply index.html]: Thank you,Done .

File: index.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  height: 90vh;
+  justify-content: center;
+}
+
+.hero__heading {
+  width: 80%;
+  font-family: 'Poppins', sans-serif;
```
Comment (Kamba56) [inline index.css]: - [Optional] I noticed you repeated the `font-family` property in multiple classes. You can consider creating a class for `font-family` and give the HTML elements the class name. For example `.poppins { font-family: 'Poppins', sans-serif; }` and then give the HTML elements the attribute `class="poppins"`. This makes your code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

File: index.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+/* hero section */
+
+#hero {
+  background-image: url(images/hero_bg.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 0 24px;
+}
```
Comment (Kamba56) [inline index.css]: - [Optional] Consider using flexbox to layout this section. This makes your work more responsive. You can use `display: flex;`, `flex-direction: column;` to layout the section.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    
+            </div>
+        </nav>
+    </header>
+    
+    <section id="hero">
+        <div class="hero">
+            <h1 class="hero__heading">
+                I’m Najash
+                Glad to see you!
```
Comment (Kamba56) [inline index.html]: - [Optional] Consider using an indentation of two spaces for readabilty of your code as this is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#html--css-best-practices) and convention.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+<body>
+    <header>
+        <nav id="nav">
+            <div class="nav">
+                <a class="nav__logo" href="#">My Logo</a>
+    
+                <img class="nav__hambuger" src="images/hambuger.svg" alt="Icon">
+    
+            </div>
+        </nav>
```
Comment (Kamba56) [inline index.html]: - [x] I think the logo is not a part of the navbar of a website so please remove the logo element from the `<nav>`.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hello @jash-debug, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Professional readme file.
- [x] All linters checks passed
- [x] Linters setup correctly
- [x] Project looks like the Figma design template

 ## Required Changes ♻️
_Please check the comments under the review for suggested changes._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [Optional] Consider using a more descriptive branch name in future projects. Branch names should describe a [feature or action](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches). This makes your work more professional.
- [Optional] It's better to follow commit message standards by making them capitalized and in the imperative form (Example: 'Modify readme file' instead of 'modified readme'). This way, your commit history will be more readable and more professional ✨

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @jash-debug, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Perfect design ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**